study.md

```diff
@@ -20,19 +20,35 @@ There is no requirement to follow any links from these pages.
 
 After doing all the required readings please tell us anything you were confused about.
 
+
+I am confused about “hoisting” and what is getting hoisted? that went over my head, I read it a few times and couldn’t figure it out.
```

Comment:
Here's a good explanation of variable hoisting.  https://en.wikipedia.org/wiki/JavaScript_syntax#Scoping_and_hoisting.